view/frontend/web/js/view/payment/method-renderer/multishipping/adyen-hpp-method.js

```diff
@@ -249,9 +249,10 @@ define([
         selectPaymentMethodType: function() {
             var self = this;
             $('#stateData').val('');
+            let paymentMethod = self.paymentMethod;
             let stateData;
-            if (!('component' in self)) {
-                let paymentMethod = self.paymentMethod;
+            if ($.trim($('#adyen-alternative-payment-container-' +
+                paymentMethod.methodIdentifier).html())==='') {
```

Comment:
What about this way? 
```suggestion
            const isEmptyContainer = $('#adyen-alternative-payment-container-' + paymentMethod.methodIdentifier).children().size() === 0;
            if (isEmptyContainer) {
```

Reply:
works as well